test/testcafe/spec/WidgetCustomization_spec.js

```diff
@@ -123,14 +123,14 @@ test.requestHooks(identifyMock)('should show custom buttons links', async t => {
       'title': 'Custom Button 1',
       'className': 'btn-customAuth-1',
       'click': function () {
-        window.location.href = 'http://www.example.com/';
+        window.location.href = 'https://www.example.com/';
```

Comment:
should we use a URL that will not load, or will load locally?

Reply:
This URL is actually fetched during our tests, so Ideally we'll add a mock for it. Interestingly... this website is down for some reason, causing our tests to fail.